roles/gluster_volume_create/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# defaults file for gluster-volume-create
+volname: glustervol
+transport: tcp
```

Comment:
Also these variable naming convention is: gluster_cluster_volume_transport... we suffix the role name, so that we do not conflict with other variables, if this role is used with other roles.